src/requestBuilder.js

```diff
@@ -60,7 +60,9 @@ var requestBuilder = {
     var numericFilters = requestBuilder._getNumericFilters(state);
     var tagFilters = requestBuilder._getTagFilters(state);
     var additionalParams = {
-      facets: facets,
+      facets: state.expandWildcardFacets
+        ? facets.filter((facet) => facet === '*' || facet[0] === '-')
```

Comment:
why is `-` (negative filters) taken into account here. It's not currently supported is it? I don't understand why this PR tries to solve that.

Reply:
is misread the [attributesToRetrieve](https://www.algolia.com/doc/api-reference/api-parameters/attributesToRetrieve/) docs as the facets doc

Reply:
maybe should the parameter just be "wildcard"?